hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1666,7 +1715,13 @@
       name: chip_csrng_lc_hw_debug_en
       desc: '''Verify the effect of LC HW debug enable on CSRNG.
 
-            TODO: This is pending SCA security review and might be removed.
+            - Initialize the OTP ctrl to boot the chip in PROD LC state (HW debug disabled).
```

Comment:
This test may have to be updated based on https://github.com/lowRISC/opentitan/issues/7140. 

I am ok to commit the test plan changes and leave a TODO pointing to the referenced issue, or updating the test case to reflect the expected behavior.